rserahlazau/07_css/Task_01_base.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD>
+        <META charset="utf-8"> 
+        <TITLE>Ссылки. Оформление</TITLE> 
+		<style type="text/css">
+            a {
+            	background-repeat: no-repeat;
+            	background-position: right center;
+            	padding-right: 20px;
+            }
+            p a[href^="https://"] {
+                background-image: url(pic/extref.png);
+                background-position: left center;
+                padding-left: 14px;
+                padding-right: 0;
+            }
+            p a[href$=".doc"] {
```

Comment:
You can combine like this for all types of files
p a[href$=".doc"],
li a[href$=".doc"]:hover {}